build.sbt

```diff
@@ -540,6 +544,84 @@ lazy val examplesWar = exampleProject("examples-war")
   )
   .dependsOn(servlet)
 
+lazy val scalafixSettings: Seq[Setting[_]] = Seq(
+  developers ++= List(
+    Developer(
+      "amarrella",
+      "Alessandro Marrella",
+      "hello@alessandromarrella.com",
+        url("https://alessandromarrella.com")
+    )
+  ),
+  addCompilerPlugin(scalafixSemanticdb),
+  scalacOptions += "-Yrangepos"
+)
+
+lazy val scalafixRules = project
+  .in(file("scalafix/rules"))
+  .settings(scalafixSettings)
+  .settings(
+    moduleName := "http4s-scalafix",
+    libraryDependencies += "ch.epfl.scala" %% "scalafix-core" % V.scalafix,
+  )
+  .enablePlugins(AutomateHeaderPlugin)
+  .disablePlugins(MimaPlugin)
+
+lazy val scalafixInput = project
+  .in(file("scalafix/input"))
+  .settings(scalafixSettings)
+  .settings(
+    skip in publish := true,
+    libraryDependencies ++= List(
+      "http4s-blaze-client",
+      "http4s-blaze-server",
+      "http4s-dsl",
+    ).map("org.http4s" %% _ % "0.21.0"),
```

Comment:
This is going to bite us the next time we add a crossbuild, but it's sufficient for today.